uint8 n_actions
int8 current_action
uint8 n_primitives
int8 current_primitive
string[] action_names
string[] ref_frames
string[] primitive_names
bool[] actions_disabled
bool[] marker_visibility
bool[] primitive_pose_editable
std_msgs/ColorRGBA[] primitive_colors
fetch_pbd_interaction/Landmark[] objects
geometry_msgs/Point[] positions
fetch_pbd_interaction/OrientationRPY[] orientations